app/src/main/java/com/keylesspalace/tusky/entity/Status.kt

```diff
@@ -30,6 +30,7 @@ data class Status(
     val account: TimelineAccount,
     @Json(name = "in_reply_to_id") val inReplyToId: String? = null,
     @Json(name = "in_reply_to_account_id") val inReplyToAccountId: String? = null,
+    val inReplyToAccount: TimelineAccount? = null,
```

Comment:
Classes in the `entity` package are for deserializing api responses, and a Mastodon status does not have this attribute. https://docs.joinmastodon.org/entities/Status/

Reply:
Ok, moved to StatusViewData.Concrete